src/vs/workbench/api/node/extHost.api.impl.ts

```diff
@@ -120,7 +120,7 @@ export function createApiFactory(
 
 	return function (extension: IExtensionDescription): typeof vscode {
 
-		if (extension.enableProposedApi && !extension.isBuiltin) {
+		if (extension.enableProposedApi && !extension.isBuiltin && extension.id !== 'ms-vscode.azure-account') {
 
```

Comment:
Not nice

Reply:
So, this is the problem. First, `vsce` prevents you from publishing an extension that uses proposed api. The nature of proposed API is that it will change, and the consequence of that is that published extensions using proposed API will be broken. That's why we don't allow/enable proposed API for published extensions. 

This leaves us with two options

1. We continue this tricky and effectively already deciding on the final API 
2. Or we just make it final API today. As said before, I think the shape of the API is OK, I do have a problem with keytar (or how it works on win/linux)

Reply:
One idea I have is to investigate on the actual WinAPI and see if we can provide a finer grained access to the credential store ourselves instead of relying on whatever keytar gives us.